lib/shopify_transporter/pipeline/magento/product/top_level_attributes.rb

```diff
@@ -14,19 +14,28 @@ def convert(hash, record)
 
           class TopLevelAttributesAccumulator < Shopify::AttributesAccumulator
             COLUMN_MAPPING = {
-              'sku' => 'sku',
               'name' => 'title',
               'description' => 'body_html',
+              'url_key' => 'handle',
+              'created_at' => 'created_at',
             }
 
             private
 
-            def input_applies?(_input)
-              true
+            def input_applies?(input)
+              true unless input['parent_id'].present?
             end
 
             def attributes_from(input)
-              map_from_key_to_val(COLUMN_MAPPING, input)
+              attributes = map_from_key_to_val(COLUMN_MAPPING, input)
+              attributes['published_scope'] = published_scope(input)
+              attributes
+            end
+
+            def published_scope(input)
+              value = input['visibility']
+              return unless value.present?
+              value == '1' ? 'web' : ' '
```

Comment:
This should check that `visibility != nil && visibility !=1` instead.

We should also add these fields if `visibility != nil && visibility !=1` :
```
Published
Published At (can be set to Updated At)
```